payments/card-payments/other-features.md

```diff
@@ -393,13 +393,13 @@ card details from the end-user a second time.
 ### Introduction
 
 * Acquirer for this service is Swedbank. You require a separate Swedbank
-acquiring number to ensure that payout transactions and regular card
-transactions are kept separate.
+  acquiring number to ensure that payout transactions and regular card
+  transactions are kept separate.
 * You need to have the 3-D Secure protocol enabled.
 * The service is available both through hosted payment pages and through direct
-API integration.
-* The current implementation is only available for gaming transactions
-(Merchant MCC: 7995).
+  API integration.
+* The current implementation is only available for gaming transactions (Merchant
+  MCC: 7995).
```

Comment:
Create an hyperlink for `MCC` pointing to `https://en.wikipedia.org/wiki/Merchant_category_code` and remove `Merchant` because the `M` in `MCC` is short for `Merchant`.

```suggestion
  [MCC][mcc]: 7995).
```